go/proxyd/backend.go

```diff
@@ -193,16 +215,51 @@ func (b *Backend) doForward(rpcReq *RPCReq) ([]byte, error) {
 		backendErrorsCtr.WithLabelValues(b.Name, rpcReq.Method).Inc()
 		return nil, wrapErr(err, "error reading response body")
 	}
+	var responseBody RPCResponse
+	err = json.Unmarshal(resB, &responseBody)
+	if err != nil {
+		return nil, wrapErr(err, "error unmarshaling response from forward")
+	}
+	go processReponse(responseBody)
 
 	return resB, nil
 }
+func processReponse(response RPCResponse) {
+	noneTooLowRegex := regexp.MustCompile(`nonce too low`)
+	gasPriceTooHighRegex := regexp.MustCompile(`gas price too high`)
+	gasPriceTooLowRegex := regexp.MustCompile(`gas price too low`)
+	invalidParametersRegex := regexp.MustCompile(`invalid parameters`)
```

Comment:
This will recompile the regexes on every request. Instead, create these regexes as variables at the  top of the file.

That said - if all we're doing is a substring match, then `strings.Contains` will likely be more efficient.